neurodamus/node.py

```diff
@@ -251,10 +251,14 @@ def __init__(self, config_file, options=None):
             config_file: A BlueConfig file
             options: A dictionary of run options typically coming from cmd line
         """
-        if config_file and config_file.endswith(".json"):
-            import libsonata
-            conf = libsonata.SimulationConfig.from_file(config_file)
-            Nd.init(log_filename=conf.output.log_file)
+        if config_file:
+            if config_file.endswith(".json"):
```

Comment:
Thanks for this PR! Super nice.

Just a small nitpick.
Since from now on we only support SONATA, maybe we should raise the ConfigurationError in case the config_file passed is "BlueConfig" and then leave the libsonata.SimulationConfig.from_file() function take care of raising an exception if the file is not correct, instead of checking for a ".json" file

Reply:
That's also an option. In this case, libsonata raises
```
RuntimeError: Path `BlueConfig` is not a file
```
Is this error msg self-explanatory ?

Reply:
But this is in the case of BlueConfig not existing right? what if it exists?

Reply:
Ah, you are right, I was testing in the wrong folder. Then in the case of existing BlueConfig 
```
RuntimeError: [json.exception.parse_error.101] parse error at line 1, column 1: syntax error while parsing value - invalid literal; last read: 'R'
```

Reply:
Exactly, thats why i thought that having a first if config_file == "BlueConfig" raise BlueConfig not supported etc
and then else let libsonata raise in case of wrong format

Reply:
I see I total misunderstood :laughing: Now I got it. Sure, I will make the change.